cadc-vos-server/src/main/java/org/opencadc/vospace/server/actions/GetNodeAction.java

```diff
@@ -147,65 +147,68 @@ public void doAction() throws Exception {
         
         if (serverNode instanceof ContainerNode) {
             ContainerNode node = (ContainerNode) serverNode;
-
             log.debug("node: " + node);
 
-            // TBD: sorting parameters
```

Comment:
Why all these deletes? This diff is impossible to grok 

If I read this correctly, the rejection of unsupported operations are indented some more so not always checked... rejection should always happen as soon as possible and not conditionally.

Reply:
The code actually changed was minimal. Didn't think to look at the diff. Yes in indentation makes it hard to read. A couple of blocks of code were moved as well which also doesn't help.

The 'if' for the indentation is checking if the caller has permission to list the children, and if they do, do the following that gives the node a iterator over its child nodes. If they don't have permission, then skip getting the child iterator. After that, write the node.